integration-tests/src/test/resources/apps/jmsservlet/src/application/JmsServlet.java

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2021, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package application;
+
+import java.io.IOException;
+import java.io.*;
```

Comment:
Could you import specific classes the servlet needs?

Reply:
done